itests/raft_messagesigner_test.go

```diff
@@ -487,12 +479,18 @@ func TestChainStoreSync(t *testing.T) {
 	}
 }
 
+func TestGoRPCAuthMultiple(t *testing.T) {
```

Comment:
Remove this. We can use count for running this several times

Reply:
oopsie